roles/teams/tasks/main.yml

```diff
@@ -57,7 +56,6 @@
   changed_when: "gsh_output.stderr != ''"
   args:
     chdir: /opt/www/grouper-shell
-  when: env == 'vm'
```

Comment:
Alright, I'll merge it then, if it gives rise to problems we should fix those there.